osquery/tables/system/darwin/account_policy_data.mm

```diff
@@ -87,13 +91,21 @@ void genAccountPolicyDataRow(const std::string& uid, Row& r) {
       TLOG << "Error parsing Account Policy data plist";
       return;
     }
+    auto isHiddenValue = [re valuesForAttribute:@"dsAttrTypeNative:IsHidden"
+                                          error:&attrErr];
+    if (isHiddenValue.count >= 1) {
+      isHidden = std::string([isHiddenValue[0] UTF8String]);
+    } else {
+      isHidden = "0";
```

Comment:
In case the value cannot be retrieved this should be set to `-1` to indicate error. Please also update documentation with this info.

Reply:
I have no issue with setting `isHidden=-1`  if the value can't be retrieved. Note the isHidden attribute only exists if it's been explicitly set, so most accounts will have have -1 in the is_hidden column.

Reply:
What happens in case `isHidden` isn't set? Is it equivalent to have it set to 0? If so then we can leave it a this.

Reply:
`isHidden` not being set is equivalent to `isHidden=0`, User account shows up normally in preferences and login screen.